### PR TITLE
tidb cloud: refine importing sample table doc

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/import-sample-data.md [hunk 1]
```diff
@@ -37,29 +35,44 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     If the region of the bucket is different from your cluster, confirm the compliance of cross region. Click **Next**.
 
-    </div>
+3. You can choose to import into pre-created tables, or import schema and data from the source.
 
-    <div label="Google Cloud">
+    - **Import into pre-created tables** allows you to create tables in TiDB in advance and select the tables that you want to import data into. In this case, you can choose up to 1000 tables to import. You can click **Chat2Query** in the left navigation pane to create tables. For more information about how to use Chat2Query, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md). **Do not choose this method when you import sample data.**
+    - **Import schema and data from S3** (This field is visible only for AWS S3) allows you to import SQL scripts for creating a table and import corresponding table data stored in S3 into TiDB.
```
Comment (Frank945946) [inline tidb-cloud/import-sample-data.md:41]:
```suggestion
    - **Import into pre-created tables** allows you to create tables in TiDB in advance and select the tables that you want to import data into. In this case, you can choose up to 1000 tables to import. You can click **Chat2Query** in the left navigation pane to create tables. For more information about how to use Chat2Query, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md). 
    - **Import schema and data from S3** (This field is visible only for AWS S3) allows you to import SQL scripts for creating a table and import corresponding table data stored in S3 into TiDB.**When you import sample data, please choose this method.**
```

File: tidb-cloud/import-sample-data.md [hunk 2]
```diff
@@ -37,29 +35,44 @@ This document describes how to import the sample data into TiDB Cloud via the UI
...
-    - **Import schema and data from S3** (This field is visible only for AWS S3) allows you to import SQL scripts for creating a table and import corresponding table data stored in S3 into TiDB.
+    - **Import into pre-created tables** allows you to create tables in TiDB in advance and select the tables that you want to import data into. In this case, you can choose up to 1000 tables to import. You can click **Chat2Query** in the left navigation pane to create tables. For more information about how to use Chat2Query, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md). **Do not choose this method when you import sample data.**
     - **Import schema and data from GCS** (This field is visible only for GCS) allows you to import SQL scripts that create a table along with its corresponding data stored in GCS directly into TiDB.
```
Comment (Frank945946) [inline tidb-cloud/import-sample-data.md:69]:
```suggestion
    - **Import into pre-created tables** allows you to create tables in TiDB in advance and select the tables that you want to import data into. In this case, you can choose up to 1000 tables to import. You can click **Chat2Query** in the left navigation pane to create tables. For more information about how to use Chat2Query, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md). 
    - **Import schema and data from GCS** (This field is visible only for GCS) allows you to import SQL scripts that create a table along with its corresponding data stored in GCS directly into TiDB.**When you import sample data, please choose this method.**
```

**Review by okJiang — COMMENTED** [review]
rest lgtm

File: tidb-cloud/import-sample-data.md [hunk 3]
```diff
@@ -37,29 +35,44 @@ This document describes how to import the sample data into TiDB Cloud via the UI
...
+
+    1. Log in to the [TiDB Cloud console](https://tidbcloud.com/) and navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
+
+        > **Tip:**
+        >
+        > If you have multiple projects, you can click <MDSvgIcon name="icon-left-projects" /> in the lower-left corner and switch to another project.
+
+    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
+
+2. Configure the source data information:
```
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:58]:
```suggestion
2. Click **Import Data** in the upper-right corner, and then fill in the following parameters:
```

File: tidb-cloud/import-sample-data.md [hunk 4]
```diff
@@ -37,29 +35,44 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     If the region of the bucket is different from your cluster, confirm the compliance of cross region. Click **Next**.
 
-    </div>
+3. You can choose to import into pre-created tables, or import schema and data from the source.
+
+    - **Import into pre-created tables** allows you to create tables in TiDB in advance and select the tables that you want to import data into. In this case, you can choose up to 1000 tables to import. You can click **Chat2Query** in the left navigation pane to create tables. For more information about how to use Chat2Query, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md). 
+    - **Import schema and data from S3** (This field is visible only for AWS S3) allows you to import SQL scripts for creating a table and import corresponding table data stored in S3 into TiDB. **When you import sample data, choose this method.**
```
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:41]:
```suggestion
    - **Import schema and data from S3** allows you to import SQL scripts for creating a table and import corresponding table data stored in S3 into TiDB.
```

File: tidb-cloud/import-sample-data.md [hunk 5]
```diff
@@ -37,29 +35,44 @@ This document describes how to import the sample data into TiDB Cloud via the UI
 
     If the region of the bucket is different from your cluster, confirm the compliance of cross region. Click **Next**.
 
-    </div>
+3. You can choose to import into pre-created tables, or import schema and data from the source.
```
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:38]:
```suggestion
3. You can choose to import into pre-created tables, or import schema and data from the source. When you import sample data, choose **Import schema and data from S3**.
```

File: tidb-cloud/import-sample-data.md [hunk 6]
```diff
@@ -37,29 +35,44 @@ This document describes how to import the sample data into TiDB Cloud via the UI
...
-    - **Import schema and data from S3** (This field is visible only for AWS S3) allows you to import SQL scripts for creating a table and import corresponding table data stored in S3 into TiDB.
-    - **Import schema and data from GCS** (This field is visible only for GCS) allows you to import SQL scripts that create a table along with its corresponding data stored in GCS directly into TiDB.
+    - **Import into pre-created tables** allows you to create tables in TiDB in advance and select the tables that you want to import data into. In this case, you can choose up to 1000 tables to import. You can click **Chat2Query** in the left navigation pane to create tables. For more information about how to use Chat2Query, see [Explore Your Data with AI-Powered Chat2Query](/tidb-cloud/explore-data-with-chat2query.md). 
+    - **Import schema and data from GCS** (This field is visible only for GCS) allows you to import SQL scripts that create a table along with its corresponding data stored in GCS directly into TiDB. **When you import sample data, choose this method.**
```
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:69]:
```suggestion
    - **Import schema and data from GCS** allows you to import SQL scripts that create a table along with its corresponding data stored in GCS directly into TiDB.
```

File: tidb-cloud/import-sample-data.md [hunk 7]
```diff
@@ -37,29 +35,44 @@ This document describes how to import the sample data into TiDB Cloud via the UI
...
     - **Bucket gsutil URI**: enter the sample data URI `gs://tidbcloud-samples-us-west1/`.
     - **Bucket Access**: you can use a GCS IAM Role to access your bucket. For more information, see [Configure GCS access](/tidb-cloud/config-s3-and-gcs-access.md#configure-gcs-access).
 
     If the region of the bucket is different from your cluster, confirm the compliance of cross region. Click **Next**.
 
-    </div>
-    </SimpleTab>
-
 3. You can choose to import into pre-created tables, or import schema and data from the source.
```
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:66]:
```suggestion
3. You can choose to import into pre-created tables, or import schema and data from the source.  When you import sample data, choose **Import schema and data from GCS**.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by okJiang — APPROVED** [review]

**Pull request merged**